src/status_im/common/scan_qr_code/view.cljs

```diff
@@ -31,7 +31,17 @@
       :size                32
       :accessibility-label :close-scan-qr-code
       :on-press            #(rf/dispatch [:navigate-back])}
-     :i/arrow-left]]
+     :i/close]
+    [quo/button
+     {:icon-only?          true
+      :type                :grey
+      :background          :blur
+      :size                32
+      :accessibility-label :show-qr-button
+      :on-press            (fn []
+                             (rf/dispatch [:navigate-back])
+                             (rf/dispatch [:open-modal :share-shell]))}
```

Comment:
Can we rename this screen to be consistent with our naming convention for screens?
`:screen/share-shell` ?